### PR TITLE
Move a piece

### PR DESCRIPTION
**Pull request by ghost**
Anna - I think I need some help with helper methods.

What is wrong with this line of code?

`<%= link_to edit_piece_path, :method=> 'GET' do %>`
`<%= image_tag '#{assign_piece}.png', :class => "#{assign_piece(piece.type, piece.color)}" %>`
`<% end %>`

I can't figure out how to make the chess piece image a link. The code above only shows a small portion of the image.

### CODE REVIEW

File: app/controllers/pieces_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    #put code that will change a piece position
+  end
+
+  def show
+
+  end
+
+  def update
+    #put code that will move a piece to a new block
+    @piece.update_attributes
```
Comment (washburnad) [inline app/controllers/pieces_controller.rb]: update attributes should take parameters.  Look back through your tutorials to see how to handle parameters

File: app/views/games/show.html.erb [hunk 2]
```diff
@@ -8,7 +8,13 @@
 					<td class="<%= color_block(row, col)%>">
 						<% @pieces.each do |piece| %>
 							<% if (piece.row_position == row) && (piece.col_position == col) %>
-								<%= link_to "", "", :class => "#{assign_piece(piece.type, piece.color)}" %>
+
+								<%= link_to edit_piece_path, :method=> 'GET' do %>
```
Comment (washburnad) [inline app/views/games/show.html.erb]: Did you try running this?  You should get an error.  Look at your routes to determine why.

File: config/routes.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@
   root 'landing_pages#index'
 
   resources :games, :only => [:new, :create, :show, :index, :update]
+  resources :pieces, :only => [:index, :show, :edit, :update]
```
Comment (washburnad) [inline config/routes.rb:7]: We shouldn't need an index action  - that would show all pieces, which we do with the game#show action

**Comment by dvanetten03** [comment]
David...are we still working on this, do you want someone else to take a shot at it or is it ready to go?  I honestly am not sure where we left it!  =)

**Comment by ghost** [comment]
So sorry Diane, I need help! Was going to email you to set up pari coding
time. I'll send a slack message to you.

**Comment by annagaiel** [comment]
:+1:

**Pull request merged**